src/PSR7/SpecFinder.php

```diff
@@ -184,10 +192,18 @@ public function findBodySpec(OperationAddress $addr) : array
     /**
      * Find the schema which describes a given response
      *
+     * @param ResponseAddress|CallbackResponseAddress $addr
+     *
      * @throws NoPath
+     * @throws NoResponseCode
```

Comment:
`NoPath` already covers `NoResponseCode` subclass

Reply:
Can you please fix at least this one and rebase PR (we have updated root namespace for the package)

Reply:
Sure, did it just now.